lms/templates/instructor/edx_ace/enrollenrolled/email/body.html

```diff
@@ -7,32 +7,66 @@
     <tr>
         <td>
             <h1>
-                {% filter force_escape %}
                 {% blocktrans %}
-                    You have been enrolled in {{ course_name }}
+
                 {% endblocktrans %}
-                {% endfilter %}
             </h1>
 
             <p style="color: rgba(0,0,0,.75);">
-                {% filter force_escape %}
-                {% blocktrans %}You have been enrolled in {{ course_name }} at {{ site_name }} by a member of the course staff. This course will now appear on your {{ site_name }} dashboard.{% endblocktrans %}
-                {% endfilter %}
                 <br />
             </p>
 
-            {% filter force_escape %}
-                {% blocktrans asvar course_cta_text %}Access the Course Materials Now{% endblocktrans %}
-            {% endfilter %}
-            {% include "ace_common/edx_ace/common/return_to_course_cta.html" with course_cta_text=course_cta_text course_cta_url=course_url %}
-
             <p style="color: rgba(0,0,0,.75);">
-                {% filter force_escape %}
                 {% blocktrans %}This email was automatically sent from {{ site_name }} to {{ full_name }}{% endblocktrans %}
-                {% endfilter %}
                 <br />
             </p>
         </td>
     </tr>
+
```

Comment:
It seems like this blank line is not required.

Reply:
There are many details in this template, this template is taken from the existing one, it's not the purpose of this PR to fix them, but to fix the main issue, however, applied.